ntbs-ui-tests/Steps/AssertSteps.cs

```diff
@@ -204,6 +224,11 @@ private string[] GetExpectedColumnsTitlesForTable(string tableId)
             };
         }
 
+        private void SetImplicitWait(TimeSpan waitTime)
+        {
+            Browser.Manage().Timeouts().ImplicitWait = waitTime;
```

Comment:
I don't really like this approach of messing with the implicit wait - at the least I think it would be good to have a comment here explaining why we're doing this and that it's tech debt to find a better way.